### PR TITLE
close #237: implemented LArray.equals(check_axes=False)

### LINKED ISSUE #237: implement larray_equiv and larray_nan_equiv
larray_equal is very picky (axes in same order and no broadcasting)

### PR DESCRIPTION
**Pull request by gdementen**
I still wonder if LArray.equiv wouldn't be better... or if check_axes=True isn't the better default value.

### CODE REVIEW

**Comment by alixdamman** [comment]
IMO, adding a `LArray.equiv()` method is definitively not a good idea. You can set check_axes=True as default value if you want.

**Comment by gdementen** [comment]
I don't really want anything. I am just unsure. I can see some users complaining either way.

**Comment by gdementen** [comment]
@alixdamman I couldn't find any mention of .equals in users' code, so I guess it won't be a problem. Ok to merge this PR as-is?

File: larray/core/array.py [hunk 1]
```diff
@@ -5081,23 +5081,27 @@ def __float__(self):
...
         atol : float or int, optional
             The absolute tolerance parameter (see Notes). Defaults to 0.
-        nans_equal: boolean, optional
+        nans_equal : boolean, optional
             Whether or not to consider nan values at the same positions in the two arrays as equal.
             By default, an array containing nan values is never equal to another array, even if that other array
             also contains nan values at the same positions. The reason is that a nan value is different from
             *anything*, including itself. Defaults to False.
+        check_axes : boolean, optional
+            Whether or not to check that the set of axes and their order is the same on both sides. Defaults to True.
```
Comment (alixdamman) [inline larray/core/array.py]: s/Defaults to True/Defaults to False

Comment (gdementen) [reply larray/core/array.py]: nice catch ! That's what you get for inverting the default at the last minute :)

Comment (alixdamman) [reply larray/core/array.py]: or working without specifications

**Pull request merged**